### DISCUSSION TITLE
Onboards - Ability to turn off live timing overlays/not display if no data available for archive races.

### DISCUSSION THREAD

**Discussion by Petarkco** [f1multiviewer/issue-tracker#17] [Feature requests] [answered]
I would like to request the ability to turn off the live timing overlay on onboard streams (via right click menu).

Also have it not display if there is no live timing data for a race. An example is the live timing overlay appearing on the onboards for the 2021 Hungarian GP showing data possibly from Baku practice?
![image](https://user-images.githubusercontent.com/25685344/173170888-1b1fd2a0-935d-47f3-9925-765919804c0e.png)

**Comment by f1multiviewer** [comment, marked as answer]
If you want you can press `t` today, to toggle the Telemetry overviews. I will also work later on replay live timing!
